src/sfctl/custom_cluster_upgrade.py

```diff
@@ -125,14 +125,16 @@ def upgrade( #pylint: disable=too-many-locals,missing-docstring,invalid-name,too
 
     client.start_cluster_upgrade(upgrade_desc, timeout=timeout)
 
-def sa_configuration_upgrade( #pylint: disable=missing-docstring,invalid-name,too-many-arguments
+def sa_configuration_upgrade( #pylint: disable=missing-docstring,invalid-name,too-many-arguments,too-many-locals
         client, cluster_config, health_check_retry='PT0H0M0S',
         health_check_wait='PT0H0M0S', health_check_stable='PT0H0M0S',
         upgrade_domain_timeout='PT0H0M0S', upgrade_timeout='PT0H0M0S',
         unhealthy_applications=0, unhealthy_nodes=0, delta_unhealthy_nodes=0,
-        upgrade_domain_delta_unhealthy_nodes=0, timeout=60):
+        upgrade_domain_delta_unhealthy_nodes=0, application_health_policies=None, timeout=60):
```

Comment:
Is this a JSON encoded list? If so you need to also add an argument parser.

Reply:
There exists a parser for it already.. parse_app_health_policy if that's what you mean